build/local_builder.go

```diff
@@ -332,6 +335,32 @@ func (l *LocalBuilder) Cleanup() (err error) {
 	return err
 }
 
+func (l *LocalBuilder) IsAllPodsRunning() bool {
+	err := infraUtils.Retry(10, 5*time.Second, func() error {
```

Comment:
I think Retry can move to `utils package` because many modules need this.

Reply:
Yeah, totally agree with you. It is quite a general one. 🥃